google_speech/__init__.py

```diff
@@ -288,7 +319,10 @@ def cl_main():
   logging.getLogger().addHandler(logging_handler)
 
   # do the job
-  Speech(args.speech, args.lang).play(args.sox_effects)
+  if args.output: # Save to file
```

Comment:
This is nitpicking, but please follow the usual PEP8 conventions, especially:
* two space before `#` for inline comments
* one space after `,` when calling functions, unpacking tuples, etc.